src/engine/engine_wb.c

```diff
@@ -97,6 +97,9 @@ static void _ocf_write_wb_complete(struct ocf_request *req, int error)
 		req->error |= error;
 	}
```

Comment:
Comments to commit message:
1. to -> too*
2. "which" seems to be out of place in this sentence: "Otherwise metadata might be flushed to early which and consequently data corruption might occur."

Reply:
done